Dockerfile

```diff
@@ -1,16 +1,32 @@
-FROM i386/debian:9
+FROM i386/debian:stable AS builder
+RUN useradd -md /opt/geneshift geneshift
+RUN apt-get update && apt-get upgrade -y
+RUN apt-get install -y curl ca-certificates
 
 ENV DUMB_INIT_VERSION=1.2.2 \
-    GENESHIFT_VERSION=1243 \
-    GOSU_VERSION=1.11
+    GENESHIFT_VERSION=1264
 
-COPY build.sh /usr/local/bin/
-RUN set -ex \
- && bash /usr/local/bin/build.sh
+RUN apt-get install curl ca-certificates -y
+
+RUN curl -sSfLo /tmp/Geneshift${GENESHIFT_VERSION}.tar.gz "https://www.geneshift.net/downloads/Geneshift${GENESHIFT_VERSION}.tar.gz"
+RUN curl -sSfLo /usr/bin/dumb-init "https://github.com/Yelp/dumb-init/releases/download/v${DUMB_INIT_VERSION}/dumb-init_${DUMB_INIT_VERSION}_amd64"
+RUN tar -vxzf "/tmp/Geneshift${GENESHIFT_VERSION}.tar.gz" -C /opt/geneshift
+RUN chmod a+x /usr/bin/dumb-init
```

Comment:
All thoose runs could be a single layer instead of 8 layers which need to be written temporarily